specification/communication/data-plane/Identity/stable/2022-06-01/CommunicationIdentity.json

```diff
@@ -0,0 +1,384 @@
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Delete an identity": {
+            "$ref": "./examples/DeleteIdentity.json"
+          }
+        }
+      }
+    },
+    "/identities/{id}/:revokeAccessTokens": {
+      "post": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Revoke all access tokens for the specific identity.",
+        "operationId": "CommunicationIdentity_RevokeAccessTokens",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Identifier of the identity.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Revoke access tokens": {
+            "$ref": "./examples/RevokeAccessTokens.json"
+          }
+        }
+      }
+    },
+    "/teamsUser/:getToken": {
+      "post": {
+        "tags": [
+          "TeamsUser"
+        ],
+        "summary": "Exchange an Azure Active Directory (Azure AD) access token of a Teams user for a new Communication Identity access token with a matching expiration time.",
+        "operationId": "CommunicationIdentity_GetTeamsUserAccessToken",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Request payload for the token exchange.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TeamsUserGetTokenRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityAccessToken"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Exchange a Teams user access token": {
+            "$ref": "./examples/GetTeamsUserAccessToken.json"
```

Comment:
Example states that an `endpoint` attribute is required but it is not documented anywhere

Reply:
it's the path of the request (discussed during the internal review)